crates/thetawave_interface/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "thetawave_interface"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = {workspace = true}
+strum = {workspace = true}
+strum_macros = {workspace = true}
+derive_more = {workspace = true}
+bevy_ecs_macros = "0.11.0"
+bevy_ecs = "0.11.0"
```

Comment:
Use consistent bevy versions across all deps. Ideally we only update 1 place when they release a new version.